misc/fibonacci_formula.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * \file
+ *
+ * \brief Implementation of [Fibonacci Sequence
+ * explicit formula solution](https://fabiandablander.com/r/Fibonacci.html)
```

Comment:
Would you like to provide a detailed description of the algorithm? You can include references from Wikipedia and other sources as long as it's not plagiarism. 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Actually, it is only based on fibonacci formula which is derived from a bunch of another formulas. So the proof is not algorithmic but mathematical. but I can surely prove it in comment if requested.

Reply:
I searched wikipedia for the proof of the algorithm but it is far more complex and none of them are using explicit formula